packages/components/table/__tests__/table.test.ts

```diff
@@ -492,6 +492,104 @@ describe('Table.vue', () => {
       wrapper.unmount()
     })
 
+    it('cell mouse enter on cell of which rowSpan > 2', async () => {
+      const wrapper = mount({
+        components: {
+          ElTable,
+          ElTableColumn,
+        },
+        template: `
+         <el-table
+          :data="testData"
+          :span-method="objectSpanMethod"
+          border
+          style="width: 100%; margin-top: 20px"
+        >
+          <el-table-column prop="id" label="ID" width="180" />
+          <el-table-column prop="name" label="Name" />
+          <el-table-column prop="amount1" label="Amount 1" />
+          <el-table-column prop="amount2" label="Amount 2" />
+          <el-table-column prop="amount3" label="Amount 3" />
+        </el-table>
+      `,
+        data() {
+          return {
+            testData: [
+              {
+                id: '12987122',
+                name: 'Tom',
+                amount1: '234',
+                amount2: '3.2',
+                amount3: 10,
+              },
+              {
+                id: '12987123',
+                name: 'Tom',
+                amount1: '165',
+                amount2: '4.43',
+                amount3: 12,
+              },
+              {
+                id: '12987124',
+                name: 'Tom',
+                amount1: '324',
+                amount2: '1.9',
+                amount3: 9,
+              },
+              {
+                id: '12987125',
+                name: 'Tom',
+                amount1: '621',
+                amount2: '2.2',
+                amount3: 17,
+              },
+              {
+                id: '12987126',
+                name: 'Tom',
+                amount1: '539',
+                amount2: '4.1',
+                amount3: 15,
+              },
+            ],
+          }
```

Comment:
Is it better to use `getTestData`?

Reply:
@FrontEndDog what happened，were these unit test failed is my fault ? see it exist in other files

Reply:
@FrontEndDog I think its ok now